Selenium.HtmlElements.Test/src/Extensions/WaitForPresentWebElementExtensionTests.cs

```diff
@@ -9,7 +9,7 @@ namespace HtmlElements.Test.Extensions
 {
     public class WaitForPresentWebElementExtensionTests : AbstractWebElementExtensionsTestFixture
     {
-        private static IEnumerable<ITestCaseData> WaitForPresentTestCases
+        private static IEnumerable<object> WaitForPresentTestCases

```

Comment:
Why change interface to object ?

Reply:
not sure it's supported in NUnit 3. I use object as given in this example:
https://github.com/nunit/docs/wiki/TestCaseSource-Attribute